org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/ProjectsManager.java

```diff
@@ -119,6 +119,13 @@
 		CREATED, CHANGED, DELETED
 	};
 
+	//@formatter:off
+	private static final List<String> buildFiles = Arrays.asList(
```

Comment:
the list of interesting build files should be provided by each project importer instance, as to minimize coupling with build systems here